src/Analyzer.JsonRuleEngine.UnitTests/ExpressionConverterTests.cs

```diff
@@ -25,6 +25,13 @@ public class ExpressionConverterTests
             .Where(property => property.Attribute != null)
             .ToDictionary(property => property.Attribute.PropertyName ?? property.Property.Name, property => property.Property, StringComparer.OrdinalIgnoreCase);
 
+        public static IReadOnlyList<object[]> InOperatorTestScenarios { get; } = new List<object[]>
```

Comment:
Add a comment above these to explain they are for the ReadJson_LeafWithValidOperator_ReturnsCorrectTypeAndValues test.